### PR TITLE
Query fix 2

### PR DESCRIPTION
**Pull request by salarchitetto**
Fixed up the trips query, removed the indexing function and just merged that into inserts function. Also added the bookeditems query and updated the user query function (thanks luis for the help)

### CODE REVIEW

File: main/sql_procedures/create_inserts.py [hunk 1]
```diff
@@ -95,7 +95,68 @@
...
+	rtl.city,
+	rtl.state,
+	rtl.country,
+	bi.source,
+	concurtrip.parsed_booked_via,
+	(CASE WHEN concurtrip.parsed_booked_via IN ('agent', 'iOS', 'Android') THEN 0
+	ELSE 
+		1 
+	END) as 'is_obt',
+	(CASE WHEN budget.id IS NOT NULL THEN 1
```
Comment (dvfreilich) [inline main/sql_procedures/create_inserts.py]: I just realized that the budgets table has only is_eligible_for_points = 1, even though I think the old captured spend one included even invalid P2Bs in captured spend.

File: main/sql_procedures/create_inserts.py [hunk 2]
```diff
@@ -95,7 +95,68 @@
...
+	ELSE
+		0
+	END) as 'is_captured'
```
Comment (dvfreilich) [inline main/sql_procedures/create_inserts.py]: Can we add `is_deployed` as well? I think it will make our lives easier in the future.

Comment (salarchitetto) [reply main/sql_procedures/create_inserts.py]: So I think the reason why the queries are different has to do with the fact that we aren't including _trip_, _organizationfact_, or _employeefact_. As for as filters go, we had two separate queries before this (captured/ deployed). Captured did include the trips table with the specific filters for trips.

Comment (salarchitetto) [reply main/sql_procedures/create_inserts.py]: I can add the is_deployed column too

**Comment by dvfreilich** [comment]
I worked through the bookeditem query with @salarchitetto, and I think this is now the correct version (but we should make sure we both agree on the changes):
`SELECT
    bi.id,
    budget.id as budget_id,
    bi.traveler_employee_id AS employee_id,
    bi.organization_id,
    bi.booker_employee_id,
    bi.cost_usd AS 'cost',
    bi.purchased_datetime,
    bi.created_at,
    bi.travel_mode_type_aggregate AS 'mode',
    bi.travel_mode_duration AS hotel_nights,
    bf.true_route_destinations,
    bh.travel_vendor,
    rtl.city,
    rtl.state,
    rtl.country,
    bi.source,
    concurtrip.parsed_booked_via,
    (CASE WHEN
     (
    concurtrip.parsed_booked_via is NULL
    OR (concurtrip.parsed_booked_via != 'agent' # removing budgets which are booked via an Agent
        AND concurtrip.parsed_booked_via != 'iOS' # removing budgets which are booked via iOS
        AND concurtrip.parsed_booked_via != 'Android' # removing budgets which are booked under android
        )
  ) THEN 1
    ELSE
        0
    END) as 'is_obt',
    (CASE WHEN budget.id IS NOT NULL THEN 1
    ELSE
        0
    END) as 'is_captured',
    (CASE WHEN bi.traveler_employee_id IS NOT NULL THEN 1
    ELSE
        0
    END) as 'is_deployed'
FROM
    rocketrip_bookings_bookeditem AS bi
    LEFT JOIN budgets AS budget ON bi.budget_id = budget.id
    LEFT JOIN rocketrip_bookings_bookedhotel AS bh ON bi.id = bh.bookeditem_ptr_id
    LEFT JOIN rocketrip_bookings_bookedflight AS bf ON bi.id = bf.bookeditem_ptr_id
    LEFT JOIN rocketripapp_rtlocation AS rtl ON rtl.id = bh.location_id
    LEFT JOIN union_table AS uni ON bi.id = uni.booked_item_id
    LEFT JOIN rocketrip_concur_bookings_concurbookedtrip AS concurtrip ON uni.concur_booked_trip_id = concurtrip.id
WHERE
    ((bi.travel_mode_type_aggregate = 'flight'
    AND bi.cost_usd < 40000)
    OR(bi.travel_mode_type_aggregate = 'rail'
        AND
        bi.cost_usd < 1000)
    OR(bi.travel_mode_type_aggregate = 'car'
        AND
        bi.cost_usd / bi.travel_mode_duration < 500)
    OR(bi.travel_mode_type_aggregate = 'hotel'
        AND
        bi.cost_usd / bi.travel_mode_duration < 2000))
    AND bi.is_in_use = 1
    AND (bi.status = 'reviewed' OR bi.status = 'pending_budget_review')
    AND bi.is_cancellation = 0
    AND bi.deleted_at IS NULL
    ;`

File: main/sql_procedures/create_inserts.py (hunk 1 above)
Comment (dvfreilich) [reply main/sql_procedures/create_inserts.py]: I discussed this with Jackie and Matt, and we think it's correct to include only `is_eligible_for_points =1` in captured spend.

File: main/sql_procedures/create_inserts.py (hunk 2 above)
Comment (dvfreilich) [reply main/sql_procedures/create_inserts.py]: `is_deployed` is in the new version of the query posted below.

**Pull request merged**